Sources/Misc/StoreKit2Setting.swift

```diff
@@ -60,6 +60,11 @@ extension StoreKit2Setting {
 
     /// - Returns: `true` iff SK2 is enabled and it's available.
     var shouldOnlyUseStoreKit2: Bool {
```

Comment:
I decided to keep this other property too because it has slightly different implications, even if it has the same implementation (right now).
This is used to determine whether to _not_ enable SK1 stuff (see #1882 for example).